motor_vehicle_report.go

```diff
@@ -0,0 +1,94 @@
+package checkr
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// MotorVehicleReport ...
+// https://docs.checkr.com/#operation/getMotorVehicleReport
+type MotorVehicleReport struct {
+	ID                    string      `json:"id,omitempty"`
+	Object                string      `json:"object,omitempty"`
+	URI                   string      `json:"uri,omitempty"`
+	Status                string      `json:"status,omitempty"`
+	CreatedAt             time.Time   `json:"created_at,omitempty"`
```

Comment:
@chandradeepak as mentioned, all `time.Time` -> `*time.Time`. And make sure all date strings are given `*time.Time` as a type.